{{ cookiecutter.__project_slug }}/README.md

```diff
@@ -30,61 +40,58 @@ dependencies are vital to your package, you will have to add them to the ``pypro
 Contribution Guide).*
 
 ## Contribution Guide
+
 {% if cookiecutter.autora_contribution_type == "theorist" -%}
 ### Theorist
 An sklearn regressor that returns an interpretable model relating experiment conditions $X$ to 
 observations $Y$.<br>
 *Example: The [Bayesian Machine Scientist](https://github.com/AutoResearch/autora-theorist-bms) (Guimerà et al., 2020, 
 in Science Advances) returns an equation governing the relationship between $X$ and $Y$.* <br>
-{% endif -%}
-{% if cookiecutter.autora_contribution_type == "experimentalist" -%}
-### Experimentalist
-A method that identifies novel experiment conditions $X'$ that yield scientific merit. 
-Experimentalists may either be implemented as a *pooler* (generating a pool of novel experiment conditions) or as a 
-sampler (selecting from an existing pool of experiment conditions $X$).<br>
+{% elif cookiecutter.autora_contribution_type == "experimentalist" -%}
+### Experimentalist {{ cookiecutter.__experimentalist_type.capitalize() }}
+A method that identifies novel experiment conditions $X'$ that yield scientific merit.
+{% if cookiecutter.__experimentalist_type == "pooler" -%} 
+*Pooler* generate a pool of novel experiment conditions.
+{% elif cookiecutter.__experimentalist_type == "sampler" -%}
+*Sampler* select from an existing pool of experiment conditions $X$).<br>
 *Example: The [Novelty Sampler](https://github.com/AutoResearch/autora-novelty-sampler) selects novel experiment 
 conditions $X'$ with respect to a pairwise distance metric applied to existing experiment conditions $X$.*
-<br>
 {% endif -%}
-{% if cookiecutter.autora_contribution_type == "experiment_runner" -%}
+{% elif cookiecutter.autora_contribution_type == "experiment_runner" -%}
 ### Experiment Runners
 A method that orchestrates the collecting of observations for a given set of 
 experiment conditions, which may include the recruitment of participants.<br>
 *Example: The [Firebase-Prolific Runner](https://github.com/AutoResearch/autora-experiment-runner-firebase-prolific) 
 enables the collection of behavioral data from human participants via web-based experiments hosted on 
 [Firebase](https://firebase.google.com/), using a pool of participants registered through
 [Prolific](https://www.prolific.co/).* <br>
-(a) **Recruitment Manager**: A method (or collection of methods) to recruit participants.<br>
+{% if cookiecutter.experiement_runner_type == "recruitment_manager" -%}
+*Recruitment Manager* is a method (or collection of methods) to recruit participants.<br>
 *Example: The [Prolific Recruitment Manager](https://github.com/AutoResearch/autora-experiment-runner-recruitment-manager-prolific)
-enables the recruitment of participants via Prolific.*<br>
-(b) **Experimentation Manager**: A method (or collection of methods) to handle the requisite experimentation processes.<br>
+enables the recruitment of participants via Prolific.*
+{% elif cookiecutter.experiement_runner_type == "participation_manager" -%}
```

Comment:
same here, are we changing the name from "Experimentation Manager" to "Participation Manager" or am i mixing things up?